Conditional Expressions/Grade_Calculation.py

```diff
@@ -0,0 +1,14 @@
+a = int(input("Enter the marks: "))
+
+if (90<a<100):

```

Comment:
it is better to use spaces like in the following:
if 90 < a < 100

Reply:
Ok sure